shaka/src/js/mse/video_element.cc

```diff
@@ -309,10 +311,20 @@ double HTMLVideoElement::Volume() const {
   return volume_;
 }
 
-void HTMLVideoElement::SetVolume(double volume) {
+ExceptionOr<void> HTMLVideoElement::SetVolume(double volume) {
+  if (volume < 0 || volume > 1) {
+    std::string message("Failed to set the 'volume' property on 'HTMLVideoElement': The volume provided (");
+    message.append(std::to_string(volume));
+    message.append(") is outside the range [0, 1].");
```

Comment:
I suggest removing the prefix here.  You can also use `util::StringPrintf` if you want:

```c++
const std::string message = util::StringPrintf(
    "The volume provided (%f) is outside the range [0, 1].", volume);
```